lib/src/base-styles/_nx-colors.scss

```diff
@@ -170,6 +186,16 @@
     // NxCounter
     --nx-color-counter: var(--nx-swatch-indigo-90);
     --nx-color-counter-active-text: var(--nx-swatch-indigo-05);
+
+    //NxProgressBar
+    --nx-color-progress-bar-text-success: var(--nx-swatch-green-60);
+    --nx-color-progress-bar-text-error: var(--nx-swatch-red-80);
+    --nx-color-progress-bar-icon-error: var(--nx-swatch-red-60);
+    --nx-color-progress-bar-counter-text-outcome: var(--nx-swatch-indigo-05);
```

Comment:
I think this is out of date with the latest mockups.  There's no longer any difference between the default `NxCounter` and the non-success/non-error `NxProgressBar` counter, in either light or dark mode.  So I don't think this variable is needed at all

Reply:
> I think this is out of date with the latest mockups. There's no longer any difference between the default `NxCounter` and the non-success/non-error `NxProgressBar` counter, in either light or dark mode. So I don't think this variable is needed at all

If I'm reading/testing this right, I believe it is still necessary, because the variable is referring to the text color of either success & error counters, not the one in progress. The non-success/non-error seems to be inheriting its styling correctly from `NxCounter`

Reply:
> I picked the name --nx-color-progress-bar-counter-text-outcome because I thought that would include "success" and "error" while excluding "in progress".

Ah I see.  `outcome` is as good a name as any there I think, though I wonder if we should just go ahead and make separate error and success text variables here even though they'd have the same value

Reply:
> > I picked the name --nx-color-progress-bar-counter-text-outcome because I thought that would include "success" and "error" while excluding "in progress".
> 
> Ah I see. `outcome` is as good a name as any there I think, though I wonder if we should just go ahead and make separate error and success text variables here even though they'd have the same value

I'm separating `--nx-color-progress-bar-counter-text-outcome` into success and error variables. That would make the purpose of each variable easier to understand. You can see the variable split here: [8ca3678](https://github.com/sonatype/sonatype-react-shared-components/pull/1049/commits/8ca3678fa51275eded1bffee6c25cd2a4ab78085)